developer_manual/core/ui-testing.rst

```diff
@@ -62,6 +69,8 @@ Set Up Test
 
 The server will bind to: ``$SRV_HOST_NAME:$SRV_HOST_PORT``.
 
+- To run the federation Sharing tests make sure you have configured HTTPS with valid certificates on both servers URLs (_`Import SSL certificates <https://doc.owncloud.org/server/10.0/admin_manual/configuration/server/import_ssl_cert.html>`_) or do not offer HTTPS at all, then the federation sharing would use HTTP.
```

Comment:
One, minor, change: s/"To run the federation Sharing tests make sure you have configured HTTPS with valid certificates on both servers URLs"/"To run the federation sharing tests make sure you have configured HTTPS with valid certificates on both servers' URLs"/g